src-common/vcastServer.ts

```diff
@@ -196,84 +196,83 @@ export async function transmitCommand(
     statusText: "",
   };
 
-  await fetch(urlToUse)
-    .then(async (response) => {
-      // the server always returns an object with exitCode and data properties
-      const rawReturnData: any = await response.json();
+  try {
+    // Await axios post
+    const response = await axios.post(urlToUse, dataAsString, {
+      headers: {
+        "Content-Type": "text/plain",
+      },
+    });
+    const rawReturnData = response.data;
 
-      if (rawReturnData.exitCode == pythonErrorCodes.internalServerError) {
-        // the error message is a list of strings, so join with \n
```

Comment:
Missing this comment

Reply:
Restored comment in c9c845a8b485173cc75c581d64e8f10b58eeb68e.